src/locales/en/en.json

```diff
@@ -7,22 +7,21 @@
   "USER_FIRST_NAME_REQUIRED": "First Name is required",
   "USER_LAST_NAME_REQUIRED": "Last Name is required",
   "USER_NAME_REQUIRED": "User Name is required",
-  "USER_PASSWORD_REQUIRED": "Password is required",
+  "USER_PASSWORD_REQUIRED": "Your password must contain a minimum of 8 characters. It must include numerals, lower and upper case alphabets and special characters, without any spaces",
   "USER_PHONE_OR_USER_EMAIL_REQUIRED": "Please enter Phone or Email to proceed",
   "USER_ROLE_REQUIRED": "Role is required",
   "USER_ORGANIZATION_NAME_REQUIRED": "Organization name is required",
   "USER_CREATE": "Create New User",
   "USER_EDIT": "Edit User",
-  "USER_FIRST_NAME_PLACEHOLDER": "Enter firstname",
-  "USER_LAST_NAME_PLACEHOLDER": "Enter lastname",
-  "USER_NAME_PLACEOLDER": "Enter user name",
-  "USER_PASSWORD_PLACEHOLDER": "Enter password",
-  "USER_EMAIL-PLACEHOLDER": "Enter user email",
-  "PHONE_NUMBER_PLACEHOLDER": "Enter Phone Number",
+  "USER_FIRST_NAME_PLACEHOLDER": "Enter Firstname",
+  "USER_LAST_NAME_PLACEHOLDER": "Enter Lastname",
+  "USER_NAME_PLACEHOLDER": "Enter User Name",
+  "USER_PASSWORD_PLACEHOLDER": "Enter Password",
+  "USER_EMAIL_PLACEHOLDER": "Enter User Email",
+  "USER_PHONE_NUMBER_PLACEHOLDER": "Enter Phone Number",
   "USER_SELECT_ROLE": "Select a Role",
   "USER_SELECT_ORGANIZATION": "Select a Organization",
-  "USER_PHONE_PLACEHOLDER":"Enter Pnone",
-  "USER_EMAIL_PLACEHOLDER":"Enter Email",
+  "USER_PHONE_PLACEHOLDER":"Enter Phone",
```

Comment:
Is this still required?

Reply:
Yes Sir, we are showing while creating user